Makefile

```diff
@@ -64,8 +60,12 @@ format-python:
 	@python3 -m black ./
 
 typecheck: typecheck-js typecheck-python
+# This is just an alias to our build script, which will perform typechecking
+# as a side-effect.
+# TODO: Do we want to have a separate typecheck command for all packages/apps?
+# Maybe using TypeScript project references?
```

Comment:
👀

Reply:
I don't know what project references are but I'm in favor of separate commands if it's easy to do. If it's a mess then I don't think it's worth the effort.

Reply:
Of course it would be good to retain a `make typecheck` or similar that covers everything.